From ba7a36b8c4cebf94b8694c20b872f44509a7d895 Mon Sep 17 00:00:00 2001
From: Tony LIU <junjie.liu@freescale.com>
Date: Thu, 22 Nov 2012 16:38:43 +0800
Subject: [PATCH 2282/2463] ENGR00234531 fix MFGTOOL issue after USB module
 loadable done

- must add a new config item to enable USB
  CONFIG_USB_FSL_ARC_OTG=y

Signed-off-by: Tony LIU <junjie.liu@freescale.com>
---
 arch/arm/configs/imx6_updater_defconfig  |    1 +
 arch/arm/configs/imx6s_updater_defconfig |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6_updater_defconfig b/arch/arm/configs/imx6_updater_defconfig
index 06ffbc5..8c58f19 100644
--- a/arch/arm/configs/imx6_updater_defconfig
+++ b/arch/arm/configs/imx6_updater_defconfig
@@ -325,6 +325,7 @@ CONFIG_MACH_MX6Q_SABREAUTO=y
 # MX6 Options:
 #
 # CONFIG_IMX_PCIE is not set
+CONFIG_USB_FSL_ARC_OTG=y
 # CONFIG_MX6_INTER_LDO_BYPASS is not set
 # CONFIG_MX6_CLK_FOR_BOOTUI_TRANS is not set
 CONFIG_ISP1504_MXC=y
diff --git a/arch/arm/configs/imx6s_updater_defconfig b/arch/arm/configs/imx6s_updater_defconfig
index bd227b3..b80f401 100644
--- a/arch/arm/configs/imx6s_updater_defconfig
+++ b/arch/arm/configs/imx6s_updater_defconfig
@@ -326,6 +326,7 @@ CONFIG_MACH_MX6Q_SABREAUTO=y
 # MX6 Options:
 #
 # CONFIG_IMX_PCIE is not set
+CONFIG_USB_FSL_ARC_OTG=y
 CONFIG_MX6_INTER_LDO_BYPASS=y
 CONFIG_ISP1504_MXC=y
 # CONFIG_MXC_IRQ_PRIOR is not set
-- 
1.7.7.4

